Client/Clique/Modules/EventList/EventListTableViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  EventListTableViewCell.swift
+//  Clique
+//
+//  Created by Infinum on 15.11.2022..
+//
+
+import Foundation
+import UIKit
+import Alamofire
+
+class EventListTableViewCell: UITableViewCell {
+    
+    @IBOutlet private var _nameLabel: UILabel!
+    @IBOutlet private var _eventImage: UIImageView!
+    @IBOutlet private var _eventLocation: UILabel!
+    @IBOutlet private var _eventDateTime: UILabel!
+    @IBOutlet private var _likeButton: UIButton!
+     
+    func configure(with event: Event) {
+        _nameLabel.text = event.eventName
+        layoutSubviews()
```

Comment:
you could make a method _setupUI() in which you would set up your UI, do not use this through layoutSubviews